packages/EasyStandard/docs/rectors/rectors.md

```diff
@@ -0,0 +1,180 @@
+---eonx_docs---
+title: 'EasyStandard: Rectors'
+weight: 2001
+---eonx_docs---
+##### [\EonX\EasyStandard\Rector\AddCoversAnnotationRector][1]
+Adds `@covers` annotation for test classes
+```php
+// before
+class SomeServiceTest extends \PHPUnit\Framework\TestCase
+{
+}
+```
+```php
+// after
+/**
+ * @covers \SomeService
+ */
+class SomeServiceTest extends \PHPUnit\Framework\TestCase
+{
+}
+```
+**Parameters**
+
+- `replaceArray` - An array of strings that will be cut from the FQCN when searching for the class covered by this test. Default value: `[]`.
```

Comment:
Personally, I would expand the FQCN acronym if it's only used once.

Reply:
Agree, done.